tests/wpt/metadata/css/css-values/vh-support-transform-translate.html.ini

```diff
@@ -0,0 +1,2 @@
+[vh-support-transform-translate.html]
+  expected: TIMEOUT
```

Comment:
It was an intermittent TIMEOUT, I guess. It's PASS in Linux and macOS though, it's still TIMEOUT in layout-2020 🤔 

```
  ▶ PASS [expected TIMEOUT] /css/css-values/vh-support-transform-translate.html
```